WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -58,6 +59,8 @@ + (NSArray *)routes
     [[FBRoute POST:@"/uiaElement/:uuid/twoFingerTap"] respondWithTarget:self action:@selector(handleTwoFingerTap:)],
     [[FBRoute POST:@"/uiaElement/:uuid/touchAndHold"] respondWithTarget:self action:@selector(handleTouchAndHold:)],
     [[FBRoute POST:@"/uiaElement/:uuid/scroll"] respondWithTarget:self action:@selector(handleScroll:)],
+    [[FBRoute POST:@"/dragfromtoforduration"] respondWithTarget:self action:@selector(handleDrag:)],
```

Comment:
I have just created it based on existing API methods examples, like "/touchAndHold" and "/doubleTap". Shouldn't we follow the same template there?